pkg/util/azureclient/mgmt/authorization/denyassignment_addons.go

```diff
@@ -30,3 +38,50 @@ func (c *denyAssignmentClient) ListForResourceGroup(ctx context.Context, resourc
 
 	return result, nil
 }
+
+func (c *denyAssignmentClient) DeleteDenyAssignment(ctx context.Context, fpTokenCredential *azidentity.ClientCertificateCredential, subscriptionDoc *api.SubscriptionDocument, doc *api.OpenShiftClusterDocument) error {
+	deleteClient, err := NewDenyAssignmentsARMClient(subscriptionDoc.ID, fpTokenCredential, nil)
+	if err != nil {
+		return err
+	}
+
+	managedResourceGroupName := strings.Split(doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, "/")[4]
```

Comment:
sorry, I'm entirely new, so just want to double check - can we be certain that the ResourceGroupID when split will have 5 or more results?